src/IW/Effects/Cabal.hs

```diff
@@ -0,0 +1,61 @@
+{- | This module contains the class definitiion of @MonadCabal@ and
+an instance of @MonadCabald@ for the @App@ monad. Instances of
+@MonadCabal@ have a @getCabalCategories@ action that returns @[Category]@
+given a @RepoOwner@ and @RepoName@.
```

Comment:
Probably it worth adding that it does so by downloading the `.cabal` file and parsing the Categories field of the file. So it's clearer what to expect from this function.

Reply:
@vrom911 Good idea.